src/odemis/acq/align/test/goffset_alignment_test.py

```
@@ -0,0 +1,125 @@
+        n_detectors = 1
+        expected = n_detectors + (n_gratings - 1)
+
+        self.assertEqual(len(res), expected)
+
+        first_grating = list(self.spgr.axes["grating"].choices.keys())[0]
+        dets_first = [d for (g, d) in res.keys() if g == first_grating]
+
+        self.assertEqual(len(dets_first), n_detectors)
+
+    @timeout(100)
+    def test_cancel(self):
```

Ruff (0.15.4)

[error] Redefinition of unused `test_cancel` from line 40: `test_cancel` redefined here

(F811)